src/main-client.ts

```diff
@@ -3348,15 +3399,40 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('sapi/v1/broker/transfer/futures', params);
   }
 
-  universalTransferBroker(
-    params: UniversalTransferBrokerParams,
-  ): Promise<BrokerSubAccount> {
+  getBrokerSubDepositHistory(
+    params: GetSubAccountDepositHistoryParams,
+  ): Promise<SubAccountDepositHistory[]> {
+    return this.getPrivate('sapi/v1/broker/subAccount/depositHist', params);
+  }
+
+  getBrokerSubAccountSpotAssets(
+    params: QuerySubAccountSpotMarginAssetInfoParams,
+  ): Promise<SubaccountBrokerSpotAssets> {
```

Comment:
```suggestion
  ): Promise<{
  data: BrokerSubAccountSpotAssets[];
  timestamp: number;
}> {
```

Something like this would be cleaner, since its just 1 level deep. same for the other methods that return { data: arr[], timestamp: number }